samples/CookieSharing/CookieSharing.WithIdentity.Core/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace CookieSharingWithIdentityCore
```

Comment:
Typically we just match the namespace with the folder, so this would be CookieSharing.WithIdentity.Core for example.  I think the nested folder is a bit redundant at this point if all 3 apps have CookieSharing in the name anyways, I'd just have all 3 live in the root like the other samples.

Reply:
Actually the folder lets you have a readme, so lets keep the folder for CookieSharing

Reply:
Except that I'd be linking them as a single "sample" of three projects. If we go with them in the root, then wouldn't I need to link them individually. Also, I have a concern about that *KeyRing* folder.

Reply:
Either way works, its not a huge deal linking them since they all start with CookieSharing, you could include the same README.md in each app for example, whichever you prefer is fine in this case...

Reply:
Let's stick with one folder for all. I'd like to have the topic have the one link ("View sample code") that lands in this folder, and the README will pop up when they arrive.